robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -265,108 +342,226 @@ public class ShadowAccessibilityNodeInfo {
-    return ((actionFlags & VISIBLE_TO_USER_MASK) != 0);
+    return ((propertyFlags & VISIBLE_TO_USER_MASK) != 0);
   }
 
   @Implementation
   public boolean isScrollable() {
-    return ((actionFlags & SCROLLABLE_MASK) != 0);
+    return ((propertyFlags & SCROLLABLE_MASK) != 0);
   }
 
   public boolean isPasteable() {
-    return ((actionFlags & PASTEABLE_MASK) != 0);
+    return ((propertyFlags & PASTEABLE_MASK) != 0);
   }
 
   @Implementation
   public boolean isEditable() {
-    return ((actionFlags & EDITABLE_MASK) != 0);
+    return ((propertyFlags & EDITABLE_MASK) != 0);
   }
-  
+
   public boolean isTextSelectionSetable() {
-    return ((actionFlags & TEXT_SELECTION_SETABLE_MASK) != 0);
+    return ((propertyFlags & TEXT_SELECTION_SETABLE_MASK) != 0);
   }
-  
+
   public boolean isA11yFocusClearable() {
-    return ((actionFlags & CLEAR_A11YFOCUSABLE_MASK) != 0);
+    return ((propertyFlags & CLEAR_A11YFOCUSABLE_MASK) != 0);
   }
-  
-  public boolean isA11yFocusable() {
-    return ((actionFlags & A11YFOCUSABLE_MASK) != 0);
+
+  @Implementation
+  public boolean isCheckable() {
+    return ((propertyFlags & CHECK_ABLE_MASK) != 0);
   }
 
-  public void setA11yFocusClearable(boolean focusClearable) {
-    actionFlags = (actionFlags & ~CLEAR_A11YFOCUSABLE_MASK) | 
-        (focusClearable ? CLEAR_A11YFOCUSABLE_MASK : 0);
+  @Implementation
+  public void setCheckable(boolean checkable) {
+    propertyFlags = (propertyFlags & ~CHECK_ABLE_MASK) |
+        (checkable ? CHECK_ABLE_MASK : 0);
   }
-  
+
+  @Implementation
+  public void setChecked(boolean checked) {
+    propertyFlags = (propertyFlags & ~CHECKED_MASK) |
+        (checked ? CHECKED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isChecked() {
+    return ((propertyFlags & CHECKED_MASK) != 0);
+  }
+
+  @Implementation
+  public void setEnabled(boolean enabled) {
+    propertyFlags = (propertyFlags & ~ENABLED_MASK) |
+        (enabled ? ENABLED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isEnabled() {
+    return ((propertyFlags & ENABLED_MASK) != 0);
+  }
+
+  @Implementation
+  public void setPassword(boolean password) {
+    propertyFlags = (propertyFlags & ~PASSWORD_MASK) |
+        (password ? PASSWORD_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isPassword() {
+    return ((propertyFlags & PASSWORD_MASK) != 0);
+  }
+
+  @Implementation
+  public void setSelected(boolean selected) {
+    propertyFlags = (propertyFlags & ~SELECTED_MASK) |
+        (selected ? SELECTED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isSelected() {
+    return ((propertyFlags & SELECTED_MASK) != 0);
+  }
+
+  @Implementation
+  public void setAccessibilityFocused(boolean focused) {
+    propertyFlags = (propertyFlags & ~A11YFOCUSED_MASK) |
+        (focused ? A11YFOCUSED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isAccessibilityFocused() {
+    return ((propertyFlags & A11YFOCUSED_MASK) != 0);
+  }
+#if ($api >= 21)
+  @Implementation
```

Comment:
Here you group setters with getters for easily excluding them based on the SDK.  For consistency with properties, let's put the setter next to each getter in this file.